feature/experimental/tunnel/otg_tests/tunnel_interface_based_ipv6_gre_encapsulation_test/tunnel_interface_based_ipv6_gre_encapsulation_test.go

```diff
@@ -0,0 +1,467 @@
+		Name:    "otgdst2",
+		IPv6:    "2001:db8:1:1:198:18:4:2",
+		IPv6Len: interfacePlen6,
+		MAC:     "00:00:03:03:03:03",
+	}
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+func TestTunnelEncapsulationByGREOverIPv6WithLoadBalance(t *testing.T) {
+	dut := ondatra.DUT(t, "dut")
+	dutPort1 := dut.Port(t, "port1")
+	dutPort2 := dut.Port(t, "port2")
+	dutPort3 := dut.Port(t, "port3")
+	ate := ondatra.ATE(t, "ate")
+	ateport1 := ate.Port(t, "port1")
+	ateport2 := ate.Port(t, "port2")
+	ateport3 := ate.Port(t, "port3")
+	egressInterfaces := []string{dutPort2.Name(), dutPort3.Name()}
+	initialEgressPkts := make([]uint64, tunnelCount)
+	initialTunnelInPkts := make([]uint64, tunnelCount)
+	initialTunnelOutPkts := make([]uint64, tunnelCount)
+	tunnelLoadblanceDiff := tunnelCount * 3
+	interfaceLoadblanceDiff := tunnelCount
+	t.Run("Configure dut with 32 tunnel interface with one ingress and 2 egress interface", func(t *testing.T) {
+		configureTunnelBaseOnDUT(t, dut, dutPort1, &dutIntf1)
+		configureTunnelBaseOnDUT(t, dut, dutPort2, &dutIntf2)
+		configureTunnelBaseOnDUT(t, dut, dutPort3, &dutIntf3)
+		step := 0
+		var overlayIPv6Nh []string
+		for unit := 0; unit < tunnelCount; unit++ {
+			tunnelSrc := incrementAddress(t, tunnelSrcIpv6Network, unit, "host")
+			tunnelDstNetwork := incrementAddress(t, tunnelDesIpv6Network, unit, "network")
+			tunnelDst := incrementAddress(t, tunnelDstNetwork, 1, "host")
+			tunnelIpv6address := incrementAddress(t, tunnelNhIpv6Network, step, "host")
+			t.Logf("unit : %d tunnel ipv6 address: %s/%d tunnel source address: %s tunnel destination: %s", unit, tunnelIpv6address, tunnelPlen6, tunnelSrc, tunnelDst)
+			if deviations.TunnelConfigPathUnsupported(dut) {
+				configureTunnelInterface(t, tunnelInterface, unit, tunnelSrc, tunnelDst, tunnelIpv6address, tunnelPlen6, dut)
+			}
+			overlayIPv6Nh = append(overlayIPv6Nh, incrementAddress(t, tunnelIpv6address, 1, "host"))
+			step = step + 2
+		}
+		t.Logf("Configure routing instance on dut")
+		configureNetworkInstance(t, dut)
+		t.Logf("Configure IPv6 tunnel destination address reachable via ECMP link")
+		underlayIpv6Nh := []string{otgIntf2.IPv6, otgIntf3.IPv6}
+		for i, nextHop := range underlayIpv6Nh {
+			_, ipv6Destination := fetchNetworkAddress(t, tunnelDesIpv6Network, 96)
+			t.Logf("configuring static route in %s destination %s with next-hop %s", dut, ipv6Destination, nextHop)
+			configStaticRoute(t, dut, ipv6Destination, nextHop, strconv.Itoa(i))
+		}
+
+		t.Logf("overlay static route via tunnel for an original IPv6 destination prefix")
+		for i, nextHop := range overlayIPv6Nh {
+			_, ipv6Destination := fetchNetworkAddress(t, encapInnerDesIpv6Network, interfacePlen6)
+			t.Logf("configuring static route in %s destination %s with next-hop %s", dut, ipv6Destination, nextHop)
+			configStaticRoute(t, dut, ipv6Destination, nextHop, strconv.Itoa(i))
+		}
+	})
+	t.Run("Configure OTG ports", func(t *testing.T) {
+		top := ate.OTG().NewConfig(t)
```

Comment:
this function is deprecated. Just use gosnappi.NewConfig()

Reply:
moved to gosnappi.NewConfig()